### PR TITLE
Improve PR template subject indication message and show how to use backticks on a PR template

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: project/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Closes #
 
 ```markdown
 ### Added
-- Added some stuff
+- Added some `method()`
```
Comment (Soullivaneuh) [inline project/.github/PULL_REQUEST_TEMPLATE.md]:
```markdown
Added `Class::classyMethod` to do great stuff
```

**Comment by Soullivaneuh** [comment]
Please make two separated commits.

cc @greg0ire :heart:

File: project/.github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (OskarStark) [inline project/.github/PULL_REQUEST_TEMPLATE.md]: 👍

**Comment by OskarStark** [comment]
Will do @Soullivaneuh

**Comment by OskarStark** [comment]
done @Soullivaneuh

**Comment by Soullivaneuh** [comment]
Commits message are not enough accurate:

1. changed sentence -> Improve PR template subject indication message
2. changed example -> Show how to use backticks on PR template

**Comment by OskarStark** [comment]
done @Soullivaneuh

File: project/.github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Closes #
 
 ```markdown
 ### Added
-- Added some stuff
+- Added some `Class::newMethod`
```
Comment (Soullivaneuh) [inline project/.github/PULL_REQUEST_TEMPLATE.md]: Well, try again:

```markdown
Added `Class::classyMethod` to do great stuff
```

:stuck_out_tongue:

**Comment by Soullivaneuh** [comment]
Thank you @OskarStark

**Pull request merged**